src/main.rs

```diff
@@ -97,22 +99,23 @@ fn main() -> ! {
         writeln!(debug_console, "oxcc is running").unwrap();
 
         if board.reset_conditions.low_power {
-            writeln!(debug_console, "WARNING: low-power reset detected").unwrap();
+            writeln!(debug_console, "WARNING: low-power reset detected")
+                .expect(DEBUG_WRITE_FAILURE);
         }
         if board.reset_conditions.window_watchdog || board.reset_conditions.independent_watchdog {
-            writeln!(debug_console, "WARNING: watchdog reset detected").unwrap();
+            writeln!(debug_console, "WARNING: watchdog reset detected").expect(DEBUG_WRITE_FAILURE);
         }
         if board.reset_conditions.software {
-            writeln!(debug_console, "WARNING: software reset detected").unwrap();
+            writeln!(debug_console, "WARNING: software reset detected").expect(DEBUG_WRITE_FAILURE);
         }
         if board.reset_conditions.por_pdr {
-            writeln!(debug_console, "WARNING: POR/PDR reset detected").unwrap();
+            writeln!(debug_console, "WARNING: POR/PDR reset detected").expect(DEBUG_WRITE_FAILURE);
```

Comment:
I think there are a few of these debugging writeln!'s in each module as well.

Reply:
Yep, will go through those dropped `Result`s separately.